river/cluster/clustream.py

```diff
@@ -0,0 +1,363 @@
+import math
+import sys
+
+from river import base, cluster
+from river.cluster.clustream_kernel import ClustreamKernel
+
+
+class Clustream(base.Clusterer):
+    """Clustream
+
+    It maintains statistical information about the data using micro-clusters.
+    These micro-clusters are temporal extensions of cluster feature vectors.
+    The micro-clusters are stored at snapshots in time following a pyramidal
+    pattern. This pattern allows to recall summary statistics from different
+    time horizons in [1]_.
+
+    Parameters
+    ----------
+    seed
+       If int, seed is the seed used by the random number generator;
+       If RandomState instance, seed is the random number generator;
+       If None, the random number generator is the RandomState instance used
+       by `np.random`.
+       It is used in the `KMeans` algorithm for the offline clustering, which is the native `KMneans` implemented in `River`.
+
+    time_window
+      The rang of the window
```

Comment:
You need to use 4 spaces to indent each parameter description, as you did for the `seed` parameter.

Reply:
Resolved.